pages/api/recipesMeals/[nameMeal].js

```diff
@@ -1,17 +1,5 @@
 import axios from 'axios';
-
-const ingredientsWithMeasure = (recipeDetails) => {
-  const keys = Object.keys(recipeDetails);
-  const keysIngredientes = keys.reduce((acc, key) => {
-    const match = key.match(/^strIngredient(\d+$)/);
-    if (match && recipeDetails[match[0]]) {
-      const index = match[1];
-      acc.push({ingredient: recipeDetails[key], mesure: recipeDetails[`strMeasure${index}`]})
-      return acc
-    } return acc
-  },[])
-  return keysIngredientes;
-}
+import { ingredientsWithMeasure } from '../../../util/functions';
 
 export default async function getRecipeDetails(req, res) {
   const { nameMeal } = req.query
```

Comment:
same comment as above, maybe we should have `api/recipesMeals?name=pie`
with a query param it is more intuitive that we are doing some filtering of recipes by `name`

Reply:
Same answer